pkg/controller/master/upgrade/upgrade_controller.go

```diff
@@ -114,3 +294,77 @@ func (h *upgradeHandler) resetLatestUpgradeLabel(latestUpgradeName string) error
 	}
 	return nil
 }
+
+func (h *upgradeHandler) upgradeKubernetes(kubernetesVersion string) error {
+	cluster, err := h.clusterCache.Get("fleet-local", "local")
+	if err != nil {
+		return err
+	}
+
+	toUpdate := cluster.DeepCopy()
+	toUpdate.Spec.KubernetesVersion = kubernetesVersion
+
+	if toUpdate.Spec.RKEConfig == nil {
+		toUpdate.Spec.RKEConfig = &provisioningv1.RKEConfig{}
+	}
+
+	drain := true
```

Comment:
The variable name looks confusing. It seems to be a constant

Reply:
Fixed in https://github.com/harvester/harvester/pull/1635/commits/c79c94c3c27d33ee40e2139fde2c76fb041090fe
I can't use a constant because I need to [pass its pointer](https://github.com/harvester/harvester/pull/1635/commits/c79c94c3c27d33ee40e2139fde2c76fb041090fe#diff-ad1688890111ebb67c1e9830de55577e7b83f1828d79f6f1f9609122cac236d3R308).